RadixWallet/Features/DappInteractionFeature/Children/Login/Coordinator/Login.swift

```diff
@@ -7,7 +7,7 @@ struct Login: Sendable, FeatureReducer {
 		let dappMetadata: DappMetadata
 		let loginRequest: P2P.Dapp.Request.AuthLoginRequestItem
 
-		var isFirstPersonaOnAnyNetwork: Bool? = nil
+		var personaPrimacy: PersonaPrimacy? = nil
```

Comment:
As nice as this sounds, it's not entirely clear what persona primacy means!

Reply:
Aha, so it's no longer a bool, but then at least put a doc comment on the type explaining what it is